extras/systemd/heketi.service

```diff
@@ -4,7 +4,7 @@ Description=Heketi Server
 [Service]
 Type=simple
 WorkingDirectory=/var/lib/heketi
-EnvironmentFile=-/etc/heketi/heketi.json
+EnvironmentFile=-/etc/heketi/heketi.env
```

Comment:
It is optional, so user may choose to create it when they want to. Non-existence of the file is not a problem as we have '-' before the file path.